tests/chainer_tests/links_tests/caffe_tests/test_caffe_function.py

```diff
@@ -920,6 +920,37 @@ def test_scale_one_bottom_with_bias(self):
         self.mock.assert_called_once_with(self.inputs[0])
 
 
+class TestSlice(TestCaffeFunctionBaseMock):
+
+    func_name = 'chainer.functions.split_axis'
+    in_shapes = [(3, 4, 3)]
+    out_shapes = [(3, 2, 3), (3, 2, 3)]
+
+    data = {
+        'layer': [
+            {
+                'name': 'slice',
```

Comment:
In this case the layer name``slice`` should be renamed ``l1`` like the other layers.